common/zac/handler.go

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
Options are parsed as part of the overall configuration which is just YAML. The options sections are parsed as `map[any]any` so as long as it fits in YAML, it can be put in the options as long as the code that reads it understands it.